### PR TITLE
[Geos] Remove empty */bin directories when statically linking

### PR DESCRIPTION
**Pull request by studiofuga**

### CODE REVIEW

**Comment by Rastaban** [comment]
It looks like your mpfr changes are also in this PR :)  I guess that will be okay as long as the mpfr changes get committed first.

**Comment by studiofuga** [comment]
Yes, I'm trying to compile a series of packages but found different issues... so I was trying to push all the changes as different PR because they are independent, but of course it didn't work.
I need to find a better way to do that. sorry for the mess.

File: ports/mpfr/portfile.cmake [hunk 1]
```diff
@@ -1,13 +1,14 @@
 include(vcpkg_common_functions)
-set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/mpfr-3.1.6)
+set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/mpfr-4.0.1)
```
Comment (KindDragon) [inline ports/mpfr/portfile.cmake]: Update version in CONTROL

Comment (Rastaban) [reply ports/mpfr/portfile.cmake]: Thanks for pointing this out.  It was addressed in PR #4964

**Pull request merged**